media-gfx/dmtx-utils/dmtx_utils-0.7.4.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A software for reading and writing Data Matrix barcodes"
+DESCRIPTION="This is open source software for reading and writing Data Matrix \
+barcodes on Linux, Unix, OS X, Windows, and mobile devices. At its core \
+libdmtx is a native shared library, allowing C/C++ programs to use \
+its capabilities without extra restrictions or overhead."
+HOMEPAGE="http://libdmtx.sourceforge.net/"
+COPYRIGHT="2009-2017 Mike Laughton"
+LICENSE="GNU LGPL v2"
```

Comment:
Should be:
```
COPYRIGHT="2006-2011 Mike Laughton and others"
LICENSE="GNU GPL v2"
```
I'm not sure, however, if it's OK to have "and others" in COPYRIGHT.
Let's see what the other mentors think about this.

BTW, I'm trying to improve a little bit the DESCRIPTION and maybe the SUMMARY too.